### PR TITLE
feat: add snaketopascal case tag

### PR DESCRIPTION
**Pull request by megatank58**

### CODE REVIEW

**Comment by almeidx** [comment]
Can you add `enum` related keywords?

**Comment by megatank58** [comment]
> Can you add `enum` related keywords?

Could you give an example?

**Review by MrMythicalYT — CHANGES_REQUESTED** [review]
for the Permissions, it is misspelled, and should probably be changed to be `PermissionFlagsBits`

File: tags/tags.toml [hunk 1]
```diff
@@ -948,3 +948,12 @@ You can remove the duplicates by resetting either the global or guild commands
...
 • Resetting guild commands: `rest.put(Routes.applicationGuildCommands(clientId, guildId), { body: [] })`
 """
+
+[case]
+keywords = ["case", "snaketopascal", "change-case", "changecase"]
+content = """
+`RangeError [BitFieldInvalid]: Invalid bitfield flag or number: undefined`
+• All `SCREAMING_SNAKE_CASE` enums have been changed to `PascalCase`
+• Intents: `Intents.FLAGS.GUILD_MESSAGES` -> `GatewayIntentBits.GuildMessages`
+• Permissions: `Permissions.FLAGS.GUILD_MESSAGES` -> `PermissionBitFields.Flags.GuildMessages`
```
Comment (almostSouji) [inline tags/tags.toml:958]:
```suggestion
• Permissions: `Permissions.FLAGS.SEND_MESSAGES` -> `PermissionFlagsBits.SendMessages`
```

**Review by almostSouji — CHANGES_REQUESTED** [review]
Should use a permission that actually exists and use PermissionFlagsBits export

**Review by almostSouji — APPROVED** [review]

**Pull request merged**